### PR TITLE
docs: reorganize integrations sidebar

### PR DESCRIPTION
**Pull request by TC-MO**
add new sub categories
- Programming
- Actor-to-Actor
- Workflows & notifications
- Data storage
- AI

add _category_.yml to each new folder to control sidebar label and position
fix all the md links to account for new structure

### CODE REVIEW

File: sources/platform/actors/development/actor_definition/index.md [hunk 1]
```diff
@@ -23,8 +23,8 @@ Actors have the following elements:
...
 
-The Apify platform provides an open [API](/api/v2), cron-style [scheduler](../../schedules), [webhooks](../../../integrations/webhooks/index.md), and [integrations](../../integrations) to services such as Zapier or Make, which make it easy for users to integrate Actors with their existing workflows. Anyone is welcome to [publish Actors](/platform/actors/publishing) in [Apify Store](https://apify.com/store), and you can even [monetize your Actors](/platform/actors/publishing/monetize).
+The Apify platform provides an open [API](/api/v2), cron-style [scheduler](../../schedules), [webhooks](../../../integrations/programming/webhooks/index.md), and [integrations](../../integrations) to services such as Zapier or Make, which make it easy for users to integrate Actors with their existing workflows. Anyone is welcome to [publish Actors](/platform/actors/publishing) in [Apify Store](https://apify.com/store), and you can even [monetize your Actors](/platform/actors/publishing/monetize).
```
Comment (jancurn) [inline sources/platform/actors/development/actor_definition/index.md:26]: The `[webhooks](../../../integrations/programming/webhooks/index.md)` link works? wow, that's cool

**Review by jancurn — APPROVED** [review]
Great, thank you

**Comment by jancurn** [comment]
Maybe just one nit: can we update also the page titles from e.g. "Keboola" to "Kebool Integration" ? See https://docs.apify.com/platform/integrations/keboola

**Comment by TC-MO** [comment]
Sure thing I will update all titles accordingly

**Pull request merged**